torchbenchmark/util/metadata_utils.py

```diff
@@ -11,6 +11,8 @@ def match_item(item_name: str, item_val: str, skip_item: Dict[str, Any]) -> bool
 
 def skip_by_metadata(test: str, device:str, jit: bool, extra_args: List[str], metadata: Dict[str, Any]) -> bool:
     "Check if the test should be skipped based on model metadata."
+    if not metadata:
```

Comment:
Can we please add the `metadata.yaml` file for this model? It is helpful for us to understand if this model uses `torch.cudnn.benchmark`, etc. And it is also helpful to setup the per-device batch sizes, skipping tests, etc.

Reply:
Fixed